src/index.web.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react'
+import { View } from 'react-native'
+import type * as native from './index'
+
+type Props = React.ComponentProps<typeof native['BlurView']>
+
+// reanimated doesn't support function components
+export class BlurView extends React.Component<Props> {
+  render() {
+    const { style, blurRadius, blurEnabled = true, blurTintColor, scale, colorTintOpacity, ...props } = this.props
+    // TODO test a good algorithm for standardizing blur() radius
+    // TODO can blurTintColor get implemented on Web?
+    // TODO can colorTintOpacity get implemented on Web?
+    // TODO can scale get implemented on Web? How would it work?
```

Comment:
The `blurTintColor` is a view beneath the blur that we can add as a child here and default its opacity to 0 and its background color to transparent, which will just use a `light` blur. Same with the colorTintOpacity, it works on the view when the tint color is set.

I included it to allow using more than `light` and `dark` which are the only two options on iOS. It allows you to pick any color and achieve a similar effects to this: 

https://github.com/candlefinance/blur-view/assets/12258850/202169d0-61d4-4a1d-a7fa-2b70361a6fc2